h/static/scripts/widget-controller.js

```diff
@@ -67,8 +114,17 @@ module.exports = function WidgetController(
   };
 
   $scope.$on(events.GROUP_FOCUSED, function () {
+    if (searchClients.length) {
+      // If the current group changes as a _result_ of loading annotations,
```

Comment:
This comment makes sense, but it's not clear how it relates to the code. Perhaps you could explain why `searchClients.length` would be zero if `the group changes because of a user action during loading of annotations`?